cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/CloudEfsAttributes.java

```diff
@@ -0,0 +1,241 @@
+        if (tags == null) {
+            tags = new HashMap<>();
+        }
+
+        tags.put(EFS_TAGKEY_NAME, efsName);
+    }
+
+    public String getCreationToken() {
+        return creationToken;
+    }
+
+    public void setCreationToken(String creationToken) {
+        this.creationToken = creationToken;
+    }
+
+    public Boolean getDeleteOnTermination() {
+        return deleteOnTermination;
+    }
+
+    public void setDeleteOnTermination(Boolean deleteOnTermination) {
+        this.deleteOnTermination = deleteOnTermination;
+    }
+
+    public Map<String, String> getTags() {
+        return tags;
+    }
+
+    public void setTags(Map<String, String> tags) {
+        this.tags = tags;
+    }
+
+    public String getPerformanceMode() {
+        return performanceMode;
+    }
+
+    public void setPerformanceMode(String performanceMode) {
+        this.performanceMode = performanceMode;
+    }
+
+    public String getThroughputMode() {
+        return throughputMode;
+    }
+
+    public void setThroughputMode(String throughputMode) {
+        this.throughputMode = throughputMode;
+    }
+
+    public Double getProvisionedThroughputInMibps() {
+        return provisionedThroughputInMibps;
+    }
+
+    public void setProvisionedThroughputInMibps(Double provisionedThroughputInMibps) {
+        this.provisionedThroughputInMibps = provisionedThroughputInMibps;
+    }
+
+    public Boolean getEncrypted() {
+        return encrypted;
+    }
+
+    public void setEncrypted(Boolean encrypted) {
+        this.encrypted = encrypted;
+    }
+
+    public String getKmsKeyId() {
+        return kmsKeyId;
+    }
+
+    public void setKmsKeyId(String kmsKeyId) {
+        this.kmsKeyId = kmsKeyId;
+    }
+
+    public String getFileSystemId() {
+        return fileSystemId;
+    }
+
+    public void setFileSystemId(String fileSystemId) {
+        this.fileSystemId = fileSystemId;
+    }
+
+    public LifeCycleState getFileState() {
+        return fileState;
+    }
+
+    public void setFileState(LifeCycleState fileState) {
+        this.fileState = fileState;
+    }
+
+    public static class Builder {
+        private String creationToken;
+
+        private Boolean deleteOnTermination;
+
+        private Map<String, String> tags;
+
+        private String performanceMode = "generalPurpose";
+
+        private String throughputMode = "bursting";
+
+        private Double provisionedThroughputInMibps;
+
+        private Boolean encrypted = true;
+
+        private String kmsKeyId;
+
+        public Builder withCreationToken(String creationToken) {
+            this.creationToken = creationToken;
+            return this;
+        }
+
+        public Builder withDeleteOnTermination(Boolean deleteOnTermination) {
+            this.deleteOnTermination = deleteOnTermination;
+            return this;
+        }
+
+        public Builder withName(String efsName) {
+            if (tags == null) {
+                tags = new HashMap<>();
```

Comment:
This is not thread-safe, could you please init at construct time?

Reply:
fixed